patcher/src/widgets/CharacterCreation/services/session/attributeOffsets.ts

```diff
@@ -43,22 +43,22 @@ export function fetchAttributeOffsetsSuccess(offsets: AttributeOffsetInfo[]) {
   };
 }
 
-export function fetchAttributeOffsetsFailed(error: ResponseError) {
```

Comment:
We should still use `ResponseError `here, I think we'd just need to add 'Message' to the class.